src/openforms/formio/tests/test_utils.py

```diff
@@ -55,3 +61,39 @@ def test_is_visible_if_not_selected(self):
         result = is_visible_in_frontend(component, data)
 
         self.assertTrue(result)
+
+    @given(
+        hidden=st.booleans(),
+        show=st.one_of(st.none(), st.booleans()),
+        when=st.one_of(st.none(), formio_component_key, st.text(max_size=0)),
+        eq=json_primitives,
+    )
+    # Sentry 326223
+    @example(hidden=True, show="", when=None, eq="")
+    def test_conditional_resiliency(
+        self,
+        hidden: bool,
+        show: None | bool,
+        when: None | str,
+        eq: JSONPrimitive,
+    ):
```

Comment:
```suggestion
    @given(
        hidden=st.booleans(),
        show=st.one_of(st.none(), st.booleans(), st.just("")),
        when=st.one_of(st.none(), formio_component_key, st.text(max_size=0)),
        eq=json_primitives,
    )
    # Sentry 326223
    @example(hidden=True, show="", when=None, eq="")
    def test_conditional_resiliency(
        self,
        hidden: bool,
        show: None | bool | str,
        when: None | str,
        eq: JSONPrimitive,
    ):
```

Following the `@example`

Reply:
ah the `st.just()` is what I couldn't find!